### PR TITLE
30409_view detail author for user

### PR DESCRIPTION
**Pull request by vietdevne**
- View all Authors
- View details author for user
- Search author with name
image
image

### CODE REVIEW

**Comment by vietdevne** [comment]
ready

**Comment by quoctd-0607** [comment]
commented
conflict code

**Comment by vietdevne** [comment]
ready

File: app/Http/Controllers/AuthorController.php [hunk 1]
```diff
@@ -13,9 +13,14 @@
...
     protected $authorRepository;
+    protected $book;
 
-    public function __construct(BaseRepositoryInterface $authorRepository){
+    public function __construct(
+        BaseRepositoryInterface $authorRepository
+        // BookRepositoryInterface $book
+    ) {
         $this->authorRepository = $authorRepository;
+        // $this->book = $book;
```
Comment (quoctd-0607) [inline app/Http/Controllers/AuthorController.php:23]: code k dùng thì bỏ

Comment (vietdevne) [reply app/Http/Controllers/AuthorController.php:23]: dạ

**Comment by vietdevne** [comment]
ready

**Pull request merged**